<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>939</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="sender">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>590</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="receiver">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>560</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Host</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="chat_box">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>520</y>
      <width>771</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="chat_window">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>681</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Poster">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>520</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="ip">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>20</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="port">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>20</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>65535</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host IP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>0</y>
      <width>49</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Host Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>50</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Folder</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="file_folder">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>150</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>xor string</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="password">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>170</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Available Files</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="avail_files">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>270</y>
      <width>231</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>520</y>
      <width>51</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="XOR">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>100</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>XOR Cipher</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ceaser">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ceaser Cipher</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="subs">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>130</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Subs Cipher</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="aes">
    <property name="geometry">
     <rect>
      <x>870</x>
      <y>130</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>AES</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ssl">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>590</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>SSL</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="rotn">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>100</y>
      <width>42</width>
      <height>25</height>
     </rect>
    </property>
    <property name="maximum">
     <number>74</number>
    </property>
    <property name="value">
     <number>13</number>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="Alpha">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>220</y>
      <width>231</width>
      <height>21</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>200</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Substitution Alphabet</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="reverse">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>130</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Reverse</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>939</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
